.reuse/dep5

```diff
@@ -9,6 +9,18 @@ Source: onnx.ai
 # Copyright: $YEAR $NAME <$CONTACT>
 # License: ...
 
+Files: .azure-pipelines/*.yml
+COPYRIGHT: Copyright (c) ONNX Project Contributors
+License: Apache-2.0
+
+
+Files: .github/**/*.md
+COPYRIGHT: Copyright (c) ONNX Project Contributors
+License: Apache-2.0
+
+Files: .github/**/*.yml
```

Comment:
BTW, some of CI related files have .yaml as extension instead... I will have a PR to update them for consistency.